packages/analytics/src/functions.ts

```diff
@@ -30,23 +30,27 @@ import { GtagCommand } from './constants';
  * @param eventName Google Analytics event name, choose from standard list or use a custom string.
  * @param eventParams Analytics event parameters.
  */
-export function logEvent(
+export async function logEvent(
```

Comment:
Does it make a difference as they all have to return a promise anyway? Changed them all for consistency in any case.

Reply:
No, it is only about consistency. You changed the signature of the functions to async but still use `then` instead of `await`. We should change them to use `await`

Reply:
Updated.